lib/mutant/config.rb

```diff
@@ -13,7 +13,11 @@ class Config
       :includes,
       :isolation,
       :jobs,
+      :kernel,
```

Comment:
@mbj not a blocker, but should this Array of Symbols be defined using `%i[...]`?